assets/js/modules/analytics/components/common/PropertySelect.js

```diff
@@ -50,10 +50,10 @@ export default function PropertySelect() {
 		return data;
 	} );
 
-	const hasExistingTag = useSelect( ( select ) => select( STORE_NAME ).hasExistingTag() );
-	const hasGTMPropertyID = useSelect( ( select ) => !! select( MODULES_TAGMANAGER ).getSingleAnalyticsPropertyID() );
 	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
 	const hasResolvedAccounts = useSelect( ( select ) => select( STORE_NAME ).hasFinishedResolution( 'getAccounts' ) );
+	const hasExistingTag = useSelect( ( select ) => select( STORE_NAME ).hasExistingTag() );
+	const hasGTMPropertyID = useSelect( ( select ) => !! select( MODULES_TAGMANAGER ).getSingleAnalyticsPropertyID() );
```

Comment:
Is there a specific reason you moved these? Just curious since I'd imagine this doesn't change anything?

Reply:
I don't remember why it has been moved. Reverted to its original state.